src/app/shared/services/http.service.ts

```diff
@@ -85,6 +85,10 @@ export class HttpService {
       .pipe(catchError(this.handleError()));
   }
 
+  getReportsInProgressThresholdTime() {
```

Comment:
Please add the return type for this method

Reply:
added return type

Reply:
A lot of methods are missing a return type. This project as a whole needs to be refactored regarding its types (or the lack there off). I think you don't need to worry about it as much as long as there is a refactoring session in sight.